documentation/assemblies/upgrading/assembly-upgrade-kafka-versions.adoc

```diff
@@ -30,6 +30,9 @@ Changing each will trigger a further rolling update.
 
 |===
 
+IMPORTANT: From Kafka {DefaultKafkaVersion}, the `log.message.format.version` is assumed to match the `inter.broker.protocol.version` and doesn't need to be set.
```

Comment:
I think this should be hardcoded since this will not change. Also, at least based on the KAfka docs, this is not the right statement.

```suggestion
IMPORTANT: From Kafka 3.0.0, when `inter.broker.protocol.version` is set to `3.0` or higher, the `log.message.format.version` option is ignored doesn't need to be set.
```

Reply:
I've updated all instances.